ui/keys/keys.go

```diff
@@ -158,3 +159,52 @@ var Keys = KeyMap{
 		key.WithHelp("q", "quit"),
 	),
 }
+
+// Rebind will update our saved keybindings from configuration values.
+func Rebind(universal, issueKeys, prKeys []config.Keybinding) error {
+	err := rebindUniversal(universal)
+	if err != nil {
+		return err
+	}
+
+	err = rebindPRKeys(prKeys)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func rebindUniversal(universal []config.Keybinding) error {
+	for _, kb := range universal {
+		log.Debug("Rebinding key", "builtin", kb.Builtin, "key", kb.Key)
```

Comment:
nice :)

Reply:
Will be adding to prs and issue keys as well.